### PR TITLE
5172 - UX/functionality improvements

### PR DESCRIPTION
**Pull request by cameron-freshworks**
*Issue #:* /bcgov/entity#5172

*Description of changes:*

* Fixed selectedEntityType bug where state wasn't being established on a re-mount after changes.
* Fixed conflicts bug where it would incorrectly see a conflict when their wasn't a new nr.
* Some style tweaks to the dialogs.
* Minor verbiage updates.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -257,6 +258,14 @@ export default class App extends Mixins(CommonMixin, DateMixin, FilingTemplateMi
     return process.env.ABOUT_TEXT
   }
 
+  /** Track changes to alteration specific pieces. */
+  private get hasAlterationChanges (): boolean {
```
Comment (severinbeauvais) [inline src/App.vue:262]: Is this used anywhere?

File: src/components/dialogs/ConfirmDialog.vue [hunk 2]
```diff
@@ -138,4 +138,9 @@ export default class ConfirmDialog extends Vue {
 </script>
 <style lang="scss" scoped>
 @import '@/assets/styles/theme.scss';
+
+.dialog-text {
+  color: $gray7 !important;
+  font-size: 1rem;
+}
```
Comment (severinbeauvais) [inline src/components/dialogs/ConfirmDialog.vue:145]: What was wrong with the original class (info-text)?

**Review by severinbeauvais — APPROVED** [review]
GTG when you're ready.

File: src/App.vue (hunk 1 above)
Comment (cameron-freshworks) [reply src/App.vue:262]: Yes, line 457 in the `goToDashboard` method.

File: src/components/dialogs/ConfirmDialog.vue (hunk 2 above)
Comment (cameron-freshworks) [reply src/components/dialogs/ConfirmDialog.vue:145]: It was being overridden, but now that i think about it. Maybe i should override v-card-text in our override. Will try that.

Comment (cameron-freshworks) [reply src/components/dialogs/ConfirmDialog.vue]: Fixed!

**Pull request merged**